cmd/README.md

```diff
@@ -10,28 +10,21 @@ Command Line tool for managing the WSO2 Micro Integrator
     1. Install [Go 1.12.x](https://golang.org/dl)
     2. Fork the [repository](https://github.com/wso2/micro-integrator)
     3. Clone your fork into any directory
-    5. `cd` into cloned directory and then cd into `micro-integrator/cmd`
-    6. Execute `go mod vendor` or `go mod download` to download all the dependencies
-
-- ### Building
-   `cd` into `micro-integrator/cmd`
-   
-    Execute `./build.sh -t micli.go -v 1.0.0 -f` to build for all platforms. (Can only be built in Mac or Linux)
-      
-    Created packages will be available at `build/target` directory
+    5. `cd` into cloned directory and then cd into `micro-integrator`
+    6. Execute `mvn clean install` to build the project
     
 - ### Running
-    Extract the compressed archive generated to a desired location.
+    `cd` into `micro-integrator/cmd/bin`
```

Comment:
Do we have to change directory to execute the CLI? Can we run from outside?

Reply:
To execute the CLI from anywhere, you need to append the location of the executable to your $PATH variable

Reply:
That's good. Shall we instruct the user to do that? Then it will look cleaner. WDYT?

Reply:
That's better, I will update the README